client/my-sites/post-type-list/index.jsx

```diff
@@ -191,16 +191,25 @@ class PostTypeList extends Component {
 	}
 
 	renderSectionHeader() {
-		const { editorUrl, postLabels } = this.props;
+		const { editorUrl, postLabels, localeSlug } = this.props;
 
 		if ( ! postLabels ) {
 			return null;
 		}
 
+		let addNewLabel = postLabels.add_new_item;
+
+		if ( 'en' === localeSlug || 'en-gb' === localeSlug ) {
+			// Temporary workaround to Sentence case label from core API for EN lang
```

Comment:
Can we maybe include a link to the Core Track ticket for extra context?

Reply:
Good idea! I'll do that with a TODO note so we know to remove it after that ships.